statick_tool/plugins/tool/cccc_tool_plugin.py

```diff
@@ -80,55 +90,85 @@ def scan(  # pylint: disable=too-many-branches
 
             logging.debug("%s", log_output)
 
-        if self.plugin_context and self.plugin_context.args.output_directory:
-            with open(self.get_name() + ".log", "wb") as flog:
-                flog.write(log_output)
+            if self.plugin_context and self.plugin_context.args.output_directory:
+                with open(self.get_name() + ".log", "wb") as flog:
```

Comment:
Since we are still running the tool once per file should we append to the log output? I did not test this.

```suggestion
                with open(self.get_name() + ".log", "ab") as flog:
```